language-support/ts/codegen/tests/ts/yarn.lock

```diff
@@ -0,0 +1,901 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
That’s a good way to ruin your negative commit stats :stuck_out_tongue:

Reply:
I suppose this file will be deleted at some point. We need to make sure it's not me doing that. ;)